test/integration/collection.js

```diff
@@ -121,6 +121,17 @@ module.exports = function(bookshelf) {
           });
       });
 
+      it('fetches a page from a relation collection without pagination', function() {
```

Comment:
Wait, now I'm confused... I thought this would only remove the `count` part from the results, not the entire `pagination` object. This needs either a more explicit name or to continue providing the `pagination` object with `page` and `pageSize`.

Reply:
Ah, I will add the pagination object in shortly. I should have though more about that.

Reply:
@ricardograca I updated it to include the pagination object

Reply:
@ricardograca Do you need anything else from me on this?

Reply:
I haven't had much time for this recently. I'll check it out later today or tomorrow. Thanks for the reminder!